src/Microsoft.VisualStudio.Editors/DesignerFramework/AccessModifierComboBox.vb

```diff
@@ -81,6 +83,7 @@ Namespace Microsoft.VisualStudio.Editors.DesignerFramework
         ' checked the project system yet)
         ' This field should only be accessed through the CustomToolsRegistered property.
         Private _customToolsRegistered As Boolean?
+        Private Shared _allowEdit As Boolean
```

Comment:
This is static right? Does this then break if you have multiple editors open at the same time?

Reply:
Good call. The variable needs to be reset afterwards

Reply:
If it's static, there'll always be a race condition in the case that multiple editors are opened at once, such when re-opening a solution. We need to find a way to make this non-static.

Reply:
Any reason we can't make it non-`Shared`?

Reply:
Made it instance. My IDE was yelling at me before? I don't know why.